charts/fluent-operator/templates/fluentbit-clusterinput-systemd.yaml

```diff
@@ -15,10 +15,15 @@ spec:
     db: /fluent-bit/tail/systemd.db
     dbSync: Normal
     systemdFilter:
+{{- if .Values.fluentbit.input.systemd.systemdFilter.enable }}
```

Comment:
I think we can remove `.Values.fluentbit.input.systemd.systemdFilter.enable` switch because if it's disabled, then all systemd logs will be collected which is too much

@karan56625

Reply:
@benjaminhuo That enable switch is added to make the `systemdFilter` optional, so that Customer can get the logs from all the services. is there any particular reason of not allowing the customer to fetch all the Systemd logs? Anyway, Systemd  is gonna read all the logs , `systemdFilter` comes top of it for filter purpose. I don't think, there will less processing if we remove all the filters. Correct me I am wrong.  As of now, we are forcing customer to get logs only from the specified services. But some customer don't want to use any filter, they just want to get the logs from all the services.` .Values.fluentbit.input.systemd.systemdFilter.enable` is for that use case. Also it is true by default, so behaviour would be same as it is now. 
let me know if that makes sense, otherwise I'll remove this flag.

Reply:
@karan56625 If the requirement is to fetch all systemd logs by disabling `systemdFilter`, then it's ok to add that switch. My concern is that it might be too many logs collected in this case when user disable this and we want users to know what exactly logs he want to collect

Reply:
Actually that's the requirement, to fetch all systemd logs by disabling systemdFilter. As of now, I don't see any option in Helm chart to achieve that. Also for customer, it is difficult to make list of expected services from which they gonna fetch the logs. If they have the list of expected service , they can just update the systemdFilter, otherwise if they want, they can fetch logs from all service. I don't see any issue there. That's an additional stuff for customer. By default, we are not fetching all the logs. but just adding option to achieve that.
Let me know, how should we conclude this. If we have disagreement on  `.Values.fluentbit.input.systemd.systemdFilter.enable`, I can remove this.

Reply:
Understood, if this is the requirements, we can add this switch